lib/author.rb

```diff
@@ -0,0 +1,21 @@
+class Author
```

Comment:
Any time you are calling class methods on a class (like `Book.new`), you should require that file.  Right now, the `write` method is working because you have required the book file in your test; but it is good practice to require it here as well (or instead of)